### PR TITLE
Add fallback page for wiki redirects

### PR DESCRIPTION
**Pull request by skyschub**
This page is for people who followed a link to the wiki we have not redirected to something new. While we plan on redirecting popular pages like the update or installation guides, there is no way we can redirect everything, so this page aims to help those who got lost.

### CODE REVIEW

**Comment by skyschub** [comment]
@goobertron Ignore the CSS, a language review on the text is more than fine. <3

File: contents/meta/404.md [hunk 1]
```diff
@@ -3,4 +3,17 @@ title: Page not found
...
-We are sorry, but the page you were looking for is nowhere to be found.
+### We are sorry, but the page you were looking for is nowhere to be found.
+{: class="text-center"}
+
+We are sorry.
+{: class="text-center"}
+
+![We are vewwy sowwy.][cat]{: class="mx-auto d-block img-fluid"}
+
+(*Photo: Mr Abdoun the scottish cat by [Mashhour][commons-cat], [CC BY-SA 3.0][cc-by-sa]*)
```
Comment (goobertron) [inline contents/meta/404.md:14]: How about Mr Wiggles, who is even sadder? https://github.com/diaspora/diaspora/blob/develop/app/assets/images/sadcat.jpg

File: contents/site/meta/wiki_sunset.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: "diaspora* Wiki Sunset"
```
Comment (goobertron) [inline contents/site/meta/wiki_sunset.md]: caps: "diaspora* wiki sunset" as per style guidelines.

File: contents/site/meta/wiki_sunset.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+subtitle: The information you requested got relocated.
```
Comment (goobertron) [inline contents/site/meta/wiki_sunset.md]: "has been" rather than "got"

File: contents/site/meta/wiki_sunset.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+<%=
+  partial "components/inline_image",
+  alt: "We are vewwy sowwy.",
+  src: static_url("site/mr_abdoun_the_scottish_cat.jpg")
+%>
+
+In Spring 2019, we decided to discontinue the diaspora\* Project Wiki and instead merge the information with the project site and the dedicated installation guides. This allows us easier control and maintenance of the project's infrastructure, as well as streamlining contributions to both the wiki and our project sites. While we attempted to keep the old URLs working by setting up redirects, we could not redirect all wiki links. If you are reading this, it means you followed a link to an old wiki resource that we could not reallocate directly. Sorry about that.
```
Comment (goobertron) [inline contents/site/meta/wiki_sunset.md]: A number of changes:

In Spring 2019, we decided to discontinue the diaspora\* project wiki and to merge the information it contained with the project site and our dedicated installation guides. This makes maintenance of the project's infrastructure far easier, as well as streamlining content. While we attempted to keep old URLs working by setting up redirects, we could not redirect all wiki articles to pages on our new site. If you are reading this, it means you followed a link to an old wiki resource that we could not reallocate directly. Sorry about that.

File: contents/site/meta/wiki_sunset.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+To help you find what you are looking for, here is a list of resources available to you:
+
+* If you need help using diaspora\*, maintaining a pod or have development related questions, check out our [guides][guides].
```
Comment (goobertron) [inline contents/site/meta/wiki_sunset.md]: Will this include the replacement for the tutorials? How about 'user guides' to distinguish it from 'installation guides' below?

I'm not 100% happy with 'user guides', so please discuss it amongst yourselves.

File: contents/site/meta/wiki_sunset.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+* If you are interested in general information on what diaspora\* is and why we are doing what we do, check out the [project website][project-site].
+* If you want to install your own pod and are looking for instructions, check out our [installation guides][install].
+
```
Comment (goobertron) [inline contents/site/meta/wiki_sunset.md:19]: How about a separate line with a link to a list of page/article titles on the new site – like the existing wiki index? That way a person might find the nearest equivalent to what they were looking for.

File: contents/site/meta/wiki_sunset.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+In the case you are unable to find what you are looking for or if you still have questions, cheeck out how you can [get help][get-help]. Sorry for the inconvenience.
```
Comment (goobertron) [inline contents/site/meta/wiki_sunset.md]: ﻿'In the case' –> 'If'; typo 'cheeck' –> 'check'.

File: contents/site/meta/wiki_sunset.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+(*Photo: Mr Abdoun the scottish cat by [Mashhour][commons-cat], [CC BY-SA 3.0][cc-by-sa]*)
```
Comment (goobertron) [inline contents/site/meta/wiki_sunset.md:22]: Mr Wiggles again?

**Review by goobertron — COMMENTED** [review]
Not much to suggest, apart from a page with an index of articles on the new site to help people find what they're looking for from the old wiki.

File: contents/meta/404.md (hunk 1 above)
Comment (skyschub) [reply contents/meta/404.md:14]: The original link in https://github.com/diaspora/diaspora/pull/3781 expired, so I no longer can validate that the image was licensed under a CC license. In addition, all other photos on that user's profile are not licensed under a CC.

File: contents/site/meta/wiki_sunset.md (hunk 5 above)
Comment (skyschub) [reply contents/site/meta/wiki_sunset.md]: We are fine with user guides for now, thanks. :)

File: contents/site/meta/wiki_sunset.md (hunk 6 above)
Comment (skyschub) [reply contents/site/meta/wiki_sunset.md:19]: Hm, no. A sitemap would actually be giant. But your question made me realized we need https://github.com/diaspora/north-star/issues/20, so let's do that.

File: contents/site/meta/wiki_sunset.md (hunk 8 above)
Comment (skyschub) [reply contents/site/meta/wiki_sunset.md:22]: See above

**Pull request merged**